install/sqls/00032-shortcode-aliases.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `aliases` (
+    `id` bigint NOT NULL,
+    `shortcode` varchar(36) NOT NULL,
+    `index` bigint NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+
+ALTER TABLE `aliases`
+    ADD PRIMARY KEY (`index`);
+
+ALTER TABLE `aliases`
+    MODIFY `index` bigint NOT NULL AUTO_INCREMENT;
+COMMIT;
```

Comment:
https://uk.wikipedia.org/wiki/%D0%A1%D0%B8%D0%BC%D0%B2%D0%BE%D0%BB_%D0%BD%D0%BE%D0%B2%D0%BE%D0%B3%D0%BE_%D1%80%D1%8F%D0%B4%D0%BA%D0%B0